Shopping List/Shopping List/Model/ShoppingModel.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ShoppingModel.swift
+//  Shopping List
+//
+//  Created by Андрей Погосский on 3/2/20.
+//  Copyright © 2020 Андрей Погосский. All rights reserved.
+//
+
+class ShoppingModel: ShoppingList {
+    private(set) var shoppingList = [String]()
+
+    func add(_ item: String) -> Bool {
+        guard shoppingList.count < 10 else { return false }
+
+        shoppingList.append(item)
+        return true
+    }
+
+    func remove(_ item: String) -> Bool {
+        guard shoppingList.contains(item) else { return false }
+
+        shoppingList.removeAll { $0 == item }
```

Comment:
Please imagine the following situation:
You have a shopping list application.

And one day you have added two items which are, literally, the same (say, `Carrot` and `Carrot`).

Then you say "Ouch!" and try to remove one of them.
According to your implementation, as a result you will remove every item `Carrot` from the list.

Is that good UX from your perspective?

Reply:
You're right, I didn't think about it. I will fix it.

Reply:
Done.